src/stores/GlobalObjectStore.tsx

```diff
@@ -27,21 +27,22 @@ class GlobalObjectStore {
     MyreviewedbyHT: 'no'
   });
 
-  addObjectToArray = async (lastarrayobject: TPrObject) => {
-    runInAction(() => {
-      this.arrayofobjects.get().push({...lastarrayobject});
-    });
-    const jsonValue = JSON.stringify(this.arrayofobjects.get());
-    await AsyncStorage.setItem('object', jsonValue);
-  };
-
   deletObjectWithId = async (objid: number) => {
     runInAction(() => {
-      this.arrayofobjects.set(
-        filter(this.arrayofobjects.get(), (c: TPrObject) => {
-          return c.Myid !== objid;
-        }),
-      );
+      filter(this.arrayofobjects.get(), (c: TSQLObject) => {
+        if (c.id === objid){
+          fetch('http://192.168.42.231:3001/api/delete', {
```

Comment:
where is the server? to check the call. add your IP to a constant file and get it from there.
all API calls should be in a store and from here we call it only